src/syscall/syscall_libc.go

```diff
@@ -140,6 +150,10 @@ func libc_getenv(name *byte) *byte
 //export read
 func libc_read(fd int32, buf *byte, count uint) int
 
+// ssize_t pread(int fd, void *buf, size_t count, off_t offset);
+//export pread
+func libc_pread(fd int32, buf *byte, count uint, offset int) int
```

Comment:
```suggestion
// ssize_t pread(int fd, void *buf, size_t count, off_t offset);
//export pread
func libc_pread(fd int32, buf *byte, count uint, offset int64) int
```

This works because:

* MacOS is always 64-bit
* Linux uses musl, which always uses a 64-bit integer for `off_t` to avoid the need for a separate pread64 function.

And this should fix the TODO above.

Reply:
Done.